identity-service/src/notifications/processNotifications/reactionNotification.js

```diff
@@ -5,44 +5,20 @@ async function processReactionNotifications (notifications, tx) {
   for (const notification of notifications) {
     const { slot, initiator: reactorId, metadata: { reaction_value: reactionValue, reacted_to: reactedTo, reaction_type: reactionType } } = notification
 
-    // First find the originating tip receive notif
-    const receiveTipNotification = await models.SolanaNotification.findOne({
+    await models.SolanaNotification.findOrCreate({
       where: {
-        type: notificationTypes.TipReceive,
+        slot,
+        type: notificationTypes.Reaction,
+        userId: receiveTipNotification.entityId, // The user receiving the reaction is the user who sent the tip, here the entityId
```

Comment:
`receiveTipNotification` doesn't exist - don't forget to pull this from `notification` as well - I believe it's `entityId`